### PR TITLE
reordering documentation

### PR DESCRIPTION
**Pull request by meori**

### CODE REVIEW

File: docs/OperatorDeployment.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+## Operator Deployment
+
+### Prerequisites
+Kubernetes 1.18+ is supported.
+
+### From script:
+```
+export OPERATOR_VERSION=v6.0.1
```
Comment (BenRub) [inline docs/OperatorDeployment.md:8]: v6.0.2

File: docs/LegacyCRD.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+## Utilizing v1beta1 CustomResourceDefinition versions
+The operator supports Kubernetes clusters from v1.13+.
```
Comment (BenRub) [inline docs/LegacyCRD.md:2]: We support K8s 1.18 and above now
we don't to support legacy CRD no more

File: docs/Main.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+[OpenShift Deployment](OpenshiftDeployment.md)
+
+
+## Full Uninstall
+
+### Uninstalling the Carbon Black Cloud Container Operator
+
+```sh
+make undeploy
```
Comment (BenRub) [inline docs/Main.md:45]: When using CNDR this is not enough
we need the bash script to be run in order to delete the `carbonblack` folder

Also,
for openshift I've add a second scc that is needed in order to remove CNDR
where is it?
https://github.com/octarinesec/octarine-operator#uninstalling-on-openshift

**Review by BenRub — CHANGES_REQUESTED** [review]
@meori

File: docs/OperatorDeployment.md (hunk 1 above)
Comment (meori) [reply docs/OperatorDeployment.md]: fixed

File: docs/LegacyCRD.md (hunk 2 above)
Comment (meori) [reply docs/LegacyCRD.md]: fixed

File: docs/Main.md (hunk 3 above)
Comment (meori) [reply docs/Main.md:45]: fixed

**Review by BenRub — APPROVED** [review]

**Pull request merged**